aws-analytics-pinpoint/src/main/java/com/amplifyframework/analytics/pinpoint/database/PinpointDatabaseHelper.kt

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.amplifyframework.analytics.pinpoint.database
+
+import android.content.Context
+import android.database.sqlite.SQLiteDatabase
+import android.database.sqlite.SQLiteOpenHelper
+
+internal class PinpointDatabaseHelper(context: Context) :
+    SQLiteOpenHelper(context, DATABASE_NAME, null, DATABASE_VERSION) {
+
+    companion object {
+        private const val DATABASE_NAME = "awspinpoint1.db"
```

Comment:
Why `pinpoint1`?  Is SDK `pinpoint`?  We would want the two to overlap so that the new library can read from the old DB, so I would expect this to be the same as the previous DB name -- not sure if it is or not

Reply:
Also these can be private vals without needing the companion object right?